rgd/geodata/serializers.py

```diff
@@ -14,7 +14,7 @@
 class SpatialEntrySerializer(serializers.ModelSerializer):
     def to_representation(self, value):
         ret = super().to_representation(value)
-        ret['footprint'] = json.loads(value.footprint.geojson)
+        # NOTE: including footprint can cause the search results to blow up in size
```

Comment:
NOTE: the default serializer no longer includes the footprint. There is a separate serializer and new endpoint to get it as some footprints can be very large